README.md

```diff
@@ -50,6 +50,33 @@ You can also build just one of the binaries in this repo by passing a name to th
 $ make build BINARIES=amtool
 ```
 
+## Docker
+
+<!--- example of complete docker run command to run a container --->
+<!--- example of a docker-compose.yml service snippet --->
+
+### Ports
+
+* `9093`: main api port
+* `9094`: cluster port 
+
+### Volumes
+
+* `/alertmanager`: storage directory
+
+These paths can also be mapped to volumes:
+* `/etc/alertmanager/alertmanager.yml`: default configuration file path
+
+### Environment variables
+
+<!--- Add documentation here --->
+
+### Command and entrypoint
+The container entrypoint is the [`alertmanager` command](https://prometheus.io/docs/alerting/configuration/)<!---More complete documentation of the command would be useful here--->. 
+
+The default docker command passed to the entrypoint is `--config.file=/etc/alertmanager/alertmanager.yml --storage.path=/alertmanager`. Specifying the docker command(using `docker run quay.io/prometheus/alertmanager $COMMAND`) thus override those default arguments.
```

Comment:
Regarding this paragraph, I'm not sure we want to get into this level of details as 1) the arguments may change over time and 2) it is more a matter of the container runtime.

Reply:
Fair enough, we could link to a documentation page for the binary.